pandas/io/parsers.py

```diff
@@ -758,15 +761,17 @@ def read_table(
     float_precision=None,
 ):
     # TODO: validation duplicated in read_csv
```

Comment:
Could you look for a way to de-duplicate the validation again? See Todo note

Reply:
I think it will be impossible because `read_table` and `read_csv` have different default values ("\t" and ","). Since `read_table` uses `read_csv` internally, I have to pass "\t" explicitly.  In that case I will explicitly provide the required `sep` value, thus validation will not work

Reply:
You could for example implement a function ``_read_csv`` called by ``read_csv`` and ``read_table`` where the default value is another parameter.

Reply:
Maybe it's better to create a validation function instead?

Reply:
I have moved validation and actual call for file reading to a separate function.